_posts/2017-01-17-announcing-afip-bill.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Announcing AfipBill"
+date: 2016-12-14 18:06:00
+categories: ["open-source", "rubygems"]
+author: "luciano"
+---
+
+If you live in Argentina and you ever use AFIP, you should already know that their platform is not the best in terms of user friendly.
+One of the things that we wanted to do for [OmbuShop](https://www.ombushop.com) was to access AFIP via [API](http://www.afip.gob.ar/ws) in order to print the bills of each seller. Unfortunately, there is no way to do this because the API doesn't have that kind of scope.
+
+<!--more-->
+
+The old way to print the bills was to access AFIP manually, create the bill and download it. As you may be thinking, this is a very tedious task, specially because we needed to send a monthly bill to each of our sellers.
```

Comment:
@lubc I'd say "... to access the AFIP website, create ... " and there is no need for "As you may be thinking" (it sounds weird and it's not necessary) 

You can just write "... download it. This is a very tedious task... "